packages/config/package.json

```diff
@@ -13,6 +13,10 @@
     ]
   },
   "dependencies": {
+    "@fuel-ui/config": "^0.10.2",
+    "@fuel-ui/css": "^0.10.2",
+    "@fuel-ui/react": "^0.10.2",
+    "@fuel-ui/test-utils": "^0.10.2",
```

Comment:
this dependency change were made because of the linking recursive, right?

the projects don't really need that. can we revert?